haskell/test/Test/Ligo/RegistryDAO.hs

```diff
@@ -146,6 +146,24 @@ test_RegistryDAO =
                baseDao (Call @"Propose") (ProposeParams proposalSize proposalMeta)
                & expectFailProposalCheck baseDao
 
+    , nettestScenarioCaps "proposal_check: fail when xtz transfer contains 0 mutez" $
```

Comment:
AFAICT this test (and the same for TreasuryDAO) doesn't actually check that the issue is solved.

Because the `min_xtz_amount` is bigger than `0` here this test will pass even if you revert the changes made in the LIGO source code.

The config needs to be changed for this test to be valid.

I suggest splitting this PR in two commits (in this order):
1. that has the test that fails (proving the check is needed in the code)
2. the commit that updates the LIGO source to fix this

Reply:
Hmm that's right, I totally miss this, I've split the test and implementation now, also make sure it throws error properly.

Reply:
Cool, works well now :ok_hand: